src/daft-connect/src/translation/logical_plan.rs

```diff
@@ -22,6 +36,32 @@ impl SparkAnalyzer<'_> {
     pub fn new(pset: &MicroPartitionSet) -> SparkAnalyzer {
         SparkAnalyzer { pset }
     }
+    pub fn create_in_memory_scan(
```

Comment:
generally speaking looks good, but we should add tets

Reply:
i agree, but since `.show` calls `print` directly, AFAIK, there's no way to capture the output to test it.

Reply:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/izEhZtMLX1KwSAhfusD5/c09a2b30-c98d-4e4b-a7cf-4edc2d5ac95f.png)

🤷 I have no idea if this AI suggestion actually works but have you tried this

Reply:
I think bare minimum we want to sure that it doesn't error